README.md

```diff
@@ -11,54 +11,104 @@ You can read more about this project on the [Kimchi book](https://o1-labs.github
 
 ## Example
 
-We assume that you already have:
-
-* `gates`: a circuit, which can be expressed as a vector of [CircuitGate](https://o1-labs.github.io/proof-systems/rustdoc/kimchi/circuits/gate/struct.CircuitGate.html)
-* a way to produce a `witness`, which can be expressed as a `[Vec<F>; COLUMNS]` (for `F` some field of your chosing)
-* `public_size`: the size of the public input
-
-Then, you can create an URS for your circuit in the following way:
-
-```rust,ignore
-use kimchi::{circuits::constraints, verifier::verify};
-use mina_curves::pasta::{fp::Fp, vesta::{Affine, VestaParameters}, pallas::Affine as Other};
-use oracle::{
-    constants::PlonkSpongeConstantsKimchi,
-    sponge::{DefaultFqSponge, DefaultFrSponge},
+The following is an example to demonstrate a full cycle workflow using circuit-construction:
+
+1. Specify a circuit
+2. Create SRS
+3. Generate prover index
+4. Generate a proof
+5. Verify the proof
+
+```rust
+use std::sync::Arc;
+use ark_ff::{FftField, PrimeField};
+use ark_poly::{EvaluationDomain, Radix2EvaluationDomain};
+use circuit_construction::{
+    *, 
```

Comment:
I think we should avoid importing `*`, unless it's a well-defined module (like `prologue`) that is meant to be opened in the root scope

Reply:
Here is an attempt to use a prologue. It does simplify the import a lot, just use one line. But I am not sure if there are any potential issues when it comes to practice :D

https://github.com/katat/kimchi/pull/1